### ISSUE TITLE
Add support for taking screenshots in main menu

### ISSUE THREAD

**Issue by ClobberXD** [minetest/minetest#7866]
##### Issue type

 - Feature request

##### Summary

To clarify: this is **not** a duplicate of #1653, but is a feature request for adding screenshot support to the mainmenu.

**Comment by Fixer-007** [comment]
Can you explain what you want exactly?

**Comment by ClobberXD** [comment]
Updated first post. Added example screenshot and improved the description.

**Comment by paramat** [comment]
Seems reasonable, but not high priority.

**Comment by ClobberXD** [comment]
IIRC, one of @rubenwardy's PRs implements this feature request, although I can't find it...

**Comment by rubenwardy** [comment]
I've closed it, but it was one of my formspec testing PRs

**Comment by ClobberXD** [comment]
Ah ok, thanks

**Comment by paramat** [comment]
> Also, builtin screen-capture capture only the contents within the window

I disagree, often screenshots are needed that show game window layout, it should be full-screen capture.

**Comment by kneekoo** [comment]
I'd like this feature too.
